Source/Events.Store/CommittedEventSequence.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Dolittle.Runtime.Events.Store
+{
+    /// <summary>
+    /// Sequence of committed events.
```

Comment:
We usually for class start the documentation with 'Represents'. And we use the <see> tag for class references where fitting 
```suggestion
    /// Represents the basis for a sequence of committed <see cref="Event" >events</see>.
```